lucille-core/src/main/java/com/kmwllc/lucille/stage/ExtractEntitiesFST.java

```diff
@@ -40,6 +44,10 @@
  *   all overlapping matches that start at that position. Defaults to false.</li>
  *   <li>stop_on_hit (Boolean, Optional) : Denotes whether this matcher should stop after one hit. Defaults to false.</li>
  *   <li>ignore_case (Boolean, Optional) : Denotes whether this Stage will ignore case determining when making matches. Defaults to false.</li>
+ *   <li>use_custom_breaks (Boolean, Optional) : If true, tokenization uses a whitespace tokenizer. Optionally, if token_break_regex is provided
```

Comment:
"use custom breaks" implies that something **custom** is going to be applied, but maybe it isn't

suggest changing this param to use_whitespace_tokenizer
mention that the default of false means "use standard tokenizer"

in constructor, should enforce that token_break_regex cannot be provided unless use_whitespace_tokenizer=true